services/app-api/src/resolvers/contract/fetchContractWithQuestions.ts

```diff
@@ -0,0 +1,144 @@
+import { GraphQLError } from 'graphql'
+import type { QueryResolvers } from '../../gen/gqlServer'
+import { NotFoundError, type Store } from '../../postgres'
+import {
+    setErrorAttributesOnActiveSpan,
+    setResolverDetailsOnActiveSpan,
+    setSuccessAttributesOnActiveSpan,
+} from '../attributeHelper'
+import { isStateUser } from '../../domain-models'
+import type { QuestionIndexType } from '../../domain-models/contractAndRates/baseContractRateTypes'
+export function fetchContractWithQuestionsResolver(
+    store: Store
+): QueryResolvers['fetchContractWithQuestions'] {
```

Comment:
Take a look at healthPlanPackageResolver, I don't think we need a separate resolver for fetchContractWithQuestions, we can just do the same thing we did in HPPResovler in ContractResovler and have it trigger the fetchQuestions when a client asks for them.